1.7/administration/upgrading.md

```diff
@@ -0,0 +1,157 @@
+---
+post_title: Upgrading
+---
+
+## Summary
+
+#### These steps provide instructions for upgrading a DC/OS cluster from DC/OS 1.6 to 1.7.  If performed on a supported OS with all prerequisites fulfilled, this upgrade _should_ preserve the state of running tasks on the cluster.  This guide follows some of the steps outlined in the Advanced DC/OS Installation Guide:
+https://dcos.io/docs/1.7/administration/installing/custom/advanced/
```

Comment:
Can you remove the `https://dcos.io` prefix from all links?  This won't render locally unless the links are relative.